src/Deprecated/Conversion/ProjectFileConverter.cs

```diff
@@ -704,7 +704,7 @@ private bool DoRepairForAssetCompat()
         {
             var toRepairImports = RequiresRepairForAssetCompat();
 
-            if (toRepairImports == null || toRepairImports.Count() == 0)
```

Comment:
In general, it probably isn't worth it to spend time on Deprecated code if it's avoidable. I'd still love for this all to disappear some day. It may be worth it to specifically opt the Deprecated folder out of any code style checks.

Reply:
Done.